ISM43362Interface.cpp

```diff
@@ -405,7 +388,7 @@ int ISM43362Interface::socket_recv(void *handle, void *data, unsigned size)
         return NSAPI_ERROR_CONNECTION_LOST;
     }
 
-    _ism.setTimeout(ISM43362_RECV_TIMEOUT);
+    _ism.setRecvTimeout(100); /* Default timeout value is 5000ms */
```

Comment:
We don't want to wait 100ms if there is no DATA. This will strongly degrade performances. that's why earlier timeout was set to 1ms.

Reply:
Also comments are values are not aligned.